src/itemAction/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { PureComponent } from 'react'
+import Item from '_utils/item'
+
+export interface ItempActionProps {
+  readonly tag?: JSX.Element
+  readonly className?: Classcat.Class
+  readonly href?: string | JSX.Element
+  readonly action?: string
+  readonly subLabel?: string
+  readonly leftAddon?: React.ReactNode
+  readonly onClick?: (event: React.MouseEvent<HTMLElement>) => void
+  readonly onBlur?: (event: React.FocusEventHandler<HTMLElement>) => void
+  readonly onFocus?: (event: React.FocusEventHandler<HTMLElement>) => void
+  readonly onMouseDown?: (event: React.MouseEvent<HTMLElement>) => void
+}
+
+class ItemAction extends PureComponent<ItempActionProps> {
```

Comment:
Shouldn't this be a functional component?

Reply:
And why is it `ItempActionProps` and not `ItemActionProps`? ;)

Reply:
Fixed thanks for the feedback ! :)